### ISSUE TITLE
Price and consumption display when hovering over bars

### ISSUE THREAD

**Issue by spacecosmos** [UtilitechAS/amsreader-firmware#692]
**Describe the solution you'd like**
A few other websites that publish electricity prices and or other form of data with bars, often have a fixed value when you hover/click the bar itself.

**Additional context**
An example can be seen here:
![Screenshot_20231208_192758_Opera](https://github.com/UtilitechAS/amsreader-firmware/assets/55928670/4f387052-4388-470e-8330-73446290dbb8)

This is a screenshot of tomorrows (Saturday) [electricity prices](https://strom.fortum.no/strompriser) from Fortum.

It would be really helpful, since the screen size of the device in use often has various forms of resolution. This way, hovering or clicking on a bar would display the value instantly.

As a side note, this would also be nice everywhere and not only on the electricity prices.

Cheers! :)

**Comment by ArnieO** [comment]
Good idea! Now, let us see if that is possible in the graphics library we're using...

**Comment by gskjold** [comment]
Test version is out, please see #1032
